picky-asn1/src/bit_string.rs

```diff
@@ -33,7 +33,7 @@ use serde::{de, ser};
 /// b.set(63, true);
 /// assert_eq!(b.is_set(63), false);
 /// ```
-#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
+#[derive(Default, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
```

Comment:
The empty `Vec` is not a valid `BitString` representation.

```suggestion
#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
```

An empty `BitString` can be constructed using `BitString::with_len(0)`.